.goreleaser.yml

```diff
@@ -20,6 +20,9 @@ builds:
     - darwin
   goarch:
     - amd64
+  hooks:
```

Comment:
Adding this to the global before hooks didn't work, because {{.Version}} wasn't evaluated and returning empty